hooks/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM amazon/dynamodb-local
```

Comment:
Ah, unfortunate that Amazon doesn't provide semver-y tags for us to use. As such I'm alright with leaving this pointing to `latest` for now.